src/Generator/Commands/ActionGenerator.php

```diff
@@ -2,18 +2,20 @@
 
 namespace Apiato\Core\Generator\Commands;
 
-use Apiato\Core\Generator\GeneratorCommand;
-use Illuminate\Support\Pluralizer;
+use Apiato\Core\Generator\FileGeneratorCommand;
+use Apiato\Core\Generator\Traits\HasTestTrait;
 use Illuminate\Support\Str;
 use Symfony\Component\Console\Input\InputOption;
 
-class ActionGenerator extends GeneratorCommand
+class ActionGenerator extends FileGeneratorCommand
 {
-    private string $model;
+    use HasTestTrait;
 
-    private string $ui;
+    protected string $model;
 
-    private string $stub;
+    protected string $stub;
+
+    protected string $ui;
 
     public static function getCommandName(): string
```

Comment:
change to use `make` instead of `generate` as command.

Reply:
Most of them are already refactored to `make`
The rest will be refactored after they are merged. (It will be easier then.)

Reply:
Done in de2ba4c2e6127447ab7e4eff8306eda135f73cfa

Reply:
this should now be ndone in all merge commands